pyfar/spatial/spatial.py

```diff
@@ -32,6 +34,16 @@ def __init__(self, sampling, round_decimals=12, center=0.0):
                     same radius.")
         super().__init__(points, radius_round, center)
 
+    def __eq__(self, other):
+        """Check for equality of two objects."""
+        return not deepdiff.DeepDiff(
+            self, other, ignore_type_in_groups=[
```

Comment:
A short comment on the ignores would be nice :)

Reply:
I deleted this, because the `ignore_type_in_groups` will be necessary for comparing with an object read from disk and thus be a requirement of  `feature-save-object`.